plat/kvm/efi.c

```diff
@@ -518,12 +518,45 @@ static inline void uk_efi_exit_bs(void)
 		uk_efi_crash("Failed to to exit Boot Services\n");
 }
 
+static inline void uk_efi_reset_attack_mitigation_enable(void)
```

Comment:
It would be useful to add a comment here with a reference to:
```
Sect 4. of TCG Platform Reset Attack Mitigation Specification Version 1.10 Rev. 17
```